src/PageVisibility.spec.js

```diff
@@ -0,0 +1,78 @@
+import { PageVisibility } from './PageVisibility';
+
+let pv;
+const event = {
+  visibilitychange: false,
+  blur: false,
+  focus: false,
+  pagehide: false,
+  pageshow: false
+};
+
+/*eslint-disable */
+const onFocus = function($event) {
+  event[$event.type] = true;
+};
+
+const onBlur = function($event) {
+  event[$event.type] = true;
+};
+/*eslint-enable */
+describe('PageVisibility', () => {
+  it('constuctor', () => {
+    pv = new PageVisibility(onFocus, onBlur);
+    expect(typeof pv.onBlur).to.equal('function');
+    expect(typeof pv.onFocus).to.equal('function');
+    expect(pv._enabled).to.be.true;
+  });
+
+  it('.enabled()', () => {
```

Comment:
Can we write the description to describe intentions a little better? (Same for all those other tests)

Reply:
Yep this was more just me trying to write the unit tests quickly and still know which function/method was failing.